app/src/main/java/social/entourage/android/entourage/category/EntourageCategoryManager.kt

```diff
@@ -86,6 +86,6 @@ object EntourageCategoryManager {
             entourageCategories[BaseEntourage.GROUPTYPE_ACTION_CONTRIBUTION] = it
         }
         //TODO make sure we have this category
-        defaultGroup = entourageCategories[BaseEntourage.GROUPTYPE_ACTION_DEMAND]!!
+        defaultGroup = entourageCategories[BaseEntourage.GROUPTYPE_ACTION_DEMAND] ?: emptyList()
```

Comment:
We should leave a crash since we can't have a empty list anyway
Maybe we can find another solution...

Reply:
Maybe we could write it like it
`entourageCategories[BaseEntourage.GROUPTYPE_ACTION_DEMAND]?.let { defaultGroup = it }`

Reply:
As you say, this list should never be null ​so we have 3 options :
- We keep !! but this may lead to a NPE crash (even it is not theoretically possible)
- We add a fallback like an empty list to ensure no exception will be thrown
- We only initialize defaultGroup if the list is not null (but Android Studio will warn that the variable is not always initialized)

--> The  fallback option is a good one to me because we get rid of any risk (and of Android Studio warnings), even if someone changes the json file that provides categories and removes GROUPTYPE_ACTION_DEMAND category